### DISCUSSION TITLE
Suggestion: More Intuitive Ordering Of Orchestra Instruments When Creating New Score Or Assigning A MuseSounds Soundfont

### DISCUSSION THREAD

**Discussion by paulalndwhr** [musescore/MuseScore#14490] [Ideas]
# Orchestra musicians might intuitively order instruments by their order in orchestra scores  from top to bottom

- I as an orchestra musician and orchestral composer am used to thinking of instruments ordered by their appearance in scores.
- my order of woodwinds is (piccolo) - flutes - oboes - (cor englais) - clarinets - (bass clarinet) - bassoons - contra bassoon and 
- my order of strings would be 1st violins - 2nd violins - violas - celli - basses

<img width="627" alt="Bildschirmfoto 2022-11-12 um 10 34 07" src="https://user-images.githubusercontent.com/109522890/201468165-a88a5b09-df0b-42a8-b55b-862a4abbc9c6.png">

# MuseSounds and MuseScore order instruments alphabetically

- alphabetical ordering feels as uncomfortable as writing on an alphabetically ordered keyboard
- inconsistency between translations: instrument order is different when working in English and German - a real bummer because selecting instruments requires carefully searching them in the full list instead of knowing where to look

<img width="1017" alt="Bildschirmfoto 2022-11-12 um 10 17 14" src="https://user-images.githubusercontent.com/109522890/201468029-4eeefbb2-20f8-42ea-b349-b07808e7c45e.png">

<img width="1440" alt="Bildschirmfoto 2022-11-12 um 10 38 07" src="https://user-images.githubusercontent.com/109522890/201468280-5143db5d-33e9-4c19-a145-59e291417b25.png">

# Solution: Follow the broadly used order of instruments in romantic era scores
- extra instruments such as saxophones in big band order on top

**Comment by ghost** [comment]
For the the New Score Wizard, at least, the instrument ordering has already been changed to follow standard score order (rather than being alphabetical), just yesterday in fact, in this PR! https://github.com/musescore/MuseScore/pull/14431
